handlers/api.go

```diff
@@ -1413,6 +1449,17 @@ func validators(queryIndices []uint64) ([]interface{}, error) {
 				eMap["performancetotal"] = eMap["performancetotal"].(int64) + currentDayIncome[uint64(validatorIndex)]
 			}
 		}
+
+		if rankCount > 0 {
+			rank7d, ok := eMap["rank7d"].(int64)
+			if !ok {
+				logger.Errorf("error converting rank7d to int64")
+				continue
+			}
+			if rank7d > 0 {
+				eMap["rankPercentage"] = (float64(rank7d) / float64(rankCount)) * 100
```

Comment:
Consider calculating the rank percentage in the query, which would absolve you from the need to have the same value in every row.

Reply:
Done that, love that!